src/webui/www/private/scripts/client.js

```diff
@@ -1681,6 +1681,20 @@ window.addEventListener("load", () => {
     window.qBittorrent.Cache.preferences.init();
     window.qBittorrent.Cache.qbtVersion.init();
 
+    // Setup color scheme switching
+    const updateColorScheme = () => {
+        const root = document.querySelector(":root");
+        const colorScheme = LocalPreferences.get("color_scheme");
+        const validScheme = (colorScheme === "light") || (colorScheme === "dark");
+        const isDark = window?.matchMedia?.("(prefers-color-scheme:dark)")?.matches ?? false;
+        root.classList.toggle("dark", ((!validScheme && isDark) || (colorScheme === "dark")));
+    };
+    if (window.matchMedia) {
```

Comment:
Isn't it always available on all browsers?
```suggestion
```
https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia#browser_compatibility

Reply:
To tell you the truth I just copied it from examples online and didn't investigate fully.
Fron the API link It seems that `matchMedia()` isn't even optional. (maybe the examples were guarding against too old browsers?).